src/wallet/wallettool.cpp

```diff
@@ -141,10 +141,13 @@ bool ExecuteWalletToolFunc(const std::string& command, const std::string& name)
             return false;
         }
         bilingual_str error;
-        if (!WalletBatch::VerifyEnvironment(path, error)) {
+        std::unique_ptr<WalletDatabase> database = CreateWalletDatabase(path);
+        if (!database->Verify(error)) {
```

Comment:
In commit "walletdb: Combine VerifyDatabaseFile and VerifyEnvironment" (ae1a9d9ebbe1b53fd245fddd1907ca90fe3a7abe)

There is a change in behavior here for the wallettool "info" and "salvage" commands. Previously these commands were just seeing if the environment could be opened here and not actually verifying anything (VerifyEnvironment method name was kind of a misnomer). Now "info" and "salvage" will be actually verifying the database before working, so they will be little slower, and maybe more useful or maybe less useful than before depending on if your database is corrupt and what you are trying to do.

Could preserve previous behavior by adding a `Database::VerifyWalletNotInUse()` method that would just call env::Open and do what the previous code used to do here. Another option for preserving previous behavior would be to call the `Database::ReloadDbEnv` method here, which should be effectively be equivalent to the previous VerifyEnvironment.

Reply:
I've added `VerifyNotInUse`. This function is still called by `Verify` so that the behavior of other `Verify` calls here won't change.

Reply:
15tNMnu84FEsYbsavhHKmQTsNBCZBPg5EY